### DISCUSSION TITLE
[BUG] [SP-API] [Issue] Product Feed submission stuck in IN_PROGRESS.

### DISCUSSION THREAD

**Discussion by ktlarc0719** [amzn/selling-partner-api-models#2802] [General]
I was using the feeds API to submit Listings feeds (Feed Type is POST_PRODUCT_DATA) and some feeds are stuck for more than 24 hrs.
I have never seen this issue for last two years, but it suddenly occurred today. Some feeds works fine, so I have no idea what is the cause. 

I have tested followings.
- Divide into small files and submit -> Not resolved.
- Re-submit same file. -> Not resolved.

Should I wait for next few days? Is anyone facing same issue?
I found similar case #1455 but looks like have not been resolved yet.

**Comment by alixleger** [comment]
Hello,

   I had the same issue last week, you should create a case to the Developper Support and ask them to cancel the feeds which are stuck in progress.

After that, I have now a new problem: Amazon process each feed in more than 15 minutes, one by one. But I noticed far less lag before...

Good luck to you!

**Comment by sungolivia** [comment]
thanks @alixleger for the recommendation, @ktlarc0719 have you already contacted our Developer Support team?
